src/camera_movement.c

```diff
@@ -0,0 +1,30 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   camera_movement.c                                  :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: gwolf <gwolf@student.42vienna.com>         +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/12/24 22:38:49 by gwolf             #+#    #+#             */
+/*   Updated: 2023/12/24 22:55:23 by gwolf            ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "camera.h"
+
+void	ft_cam_move_forward(t_cam *cam, float distance)
```

Comment:
Is distance not a constant value set by us/in the config file? Could just use a macro and give the direction with a bool.

Reply:
Right now yes. 

The idea is to at least be able to somewhat influence the value. 
E.g. in my fdf I had the option to set the incrementer to min/mid/max. If I then zoom, it would add little/somewhat/much to the zoom value. Same thing could be applied here: right now we only add 1. We could influence this value to make the user experience more satisfying.

Reply:
You mean one key which increases and another which decreases movement speed?

distance = MACRO_VALUE - INCREMENT x keystroke

Sth like that?

Reply:
Pass as distance a variable which is manipulated by a keypress.

I would call it like ft_cam_move_forward(&render->cam, render->increment), where increment can be manipulated by a keypress to be either 1, 2 or 3 or smth